addons/web/static/src/js/core/ajax.js

```diff
@@ -433,15 +433,56 @@ var loadXML = (function () {
     };
 })();
 
-return {
+
+/**
+ * Loads the given js and css libraries. Note that the ajax loadJS and loadCSS methods
+ * don't do anything if the given file is already loaded.
+ *
+ * @param {Object} libs
+ * @param {Array<string> | Array<Array<string>>} [libs.jsLibs=[]] The list of JS files
+ *   that we want to load.  If it is a list of strings, each file will be loaded
+ *   in parallel.  If it is a list of lists, the inner lists will be loaded
+ *   sequentially, and each element of the main list will be loaded in parallel.
+ * @param {Array<string>} [libs.cssLibs=[]] A list of css files, to be loaded in
```

Comment:
@param {Array<string> | Array<Array<string>>} [libs.jsLibs=[]] The list of JS files
that we want to load.  The list may contain strings (the files to load), or lists of
strings. The first level is loaded sequentially, and files listed in inner lists are loaded
in parallel.